### ISSUE TITLE
Is "lutro.graphics.scale" now working ?

### ISSUE THREAD

**Issue by VinZombie** [libretro/libretro-lutro#199]
Hello, i'm trying to use the lutro.graphics.scale(scaleX, scaleY) function. Is it implemented ? Thank you in advance.

**Comment by kivutar** [comment]
Hi, it's a stub, not working for now.

**Comment by VinZombie** [comment]
Thank you for the answer.
Is the function possible to implement, or do I get rid of it ?
Do I have to close this issue ?

**Comment by VinZombie** [comment]
(Sorry for the bad english, I'm french)

**Comment by kivutar** [comment]
I'm French too so don't worry.

I think it's possible to implement it. It might be slow and work only for integer scaling, which is clearly not optimal.

By the way, what are you trying to scale? If it's the whole screen, there might be some other ways to handle things.

**Comment by VinZombie** [comment]
Oh ! Only integers and slow function.... If I understand correctly, it is because Lutro is only software rendered ?
Yes, it's the all screen that I try to scale. It's optional, but i want to zoom out and in  slowly from a 2d isometric map made of .png and .jpg  files. I don't have any idea of how to do it without a scale function...
Do you have a solution ?

**Comment by RobLoach** [comment]
Correct, Lutro is software rendered... You could scale things manually yourself?

```
love.graphics.draw(drawable, x, y, r, sx, sy, ox, oy, kx, ky)
```

I see some stuff in the painter about scaling. Haven't tested it.

**Comment by VinZombie** [comment]
I have tested, i don't succed to make it work with
> love.gaphics.draw(...)
Add the scale values have no effect. (Rotation doesn't have effect neither).
I tried with official version from Retroarch, compiled versions on Linux and WIndows. No success.

**Comment by kivutar** [comment]
I also see some scaling stuff in painter which sets these two variables `sx` and `sy`.
But I don't see `p->trans->sx` used in any of the drawing functions.

**Comment by VinZombie** [comment]
@RobLoach , @kivutar , thank you for investigate !
One question, do you think it could be possible to scale with float each sprite ? Or is it limited like global scale ?

**Comment by kivutar** [comment]
I think it should be, stuff like SDL allow this right?
